app/build.gradle

```diff
@@ -353,7 +353,7 @@ distributions {
                 exclude 'es.jar'
                 exclude 'es-*.jar'
             }
-            into("lib") {
+            into("lib/main") {
```

Comment:
The JarHell test now checks for duplicate paths in the classpath. We add the main jar twice, 1) as first element in path 2) thorugh `lib/*`. The changes only add the main jar at first position.